lib/helpers/contentfulEntry.js

```diff
@@ -138,10 +138,13 @@ async function getTopicTemplates(contentfulEntry) {
 
   const templateNames = Object.keys(templatesConfig);
   const promises = templateNames.map((templateName) => {
```

Comment:
Minor observation. Using [Promise.map](http://bluebirdjs.com/docs/api/promise.map.html#promise.map) here could condense this code a bit more.